nbconvert/templates/latex/base.tplx

```diff
@@ -20,6 +20,8 @@ This template does not define a docclass, the inheriting class must define this.
     % Basic figure setup, for now with no caption control since it's done
     % automatically by Pandoc (which extracts ![](path) syntax from Markdown).
     \usepackage{graphicx}
+    % protect against spaces in asset names
+    \usepackage[space]{grffile}
```

Comment:
Only sort of works, but it keeps things from hard crashing and just introduces rendering idiosyncrasies for when any code paths might leave a space in a rendered asset path.

Reply:
This seems like it's redundant with the addition of grffile below.

Reply:
grffile only protects against spaces in names when using pdflatex, but not XeLaTeX.

Reply:
In the example I provided at https://github.com/mpacer/tex_ex/ `grffile` protects against spaces in names when using XeLaTeX. But, it requires adding the fix from https://github.com/ho-tex/oberdiek/issues/31#issuecomment-441094438 to get it to work.